styles.css

```diff
@@ -333,3 +333,118 @@ a:active {
   color: #c1c7d0;
   opacity: 1;
 }
+
+.contact-me {
+  display: flex;
+  flex-flow: column wrap;
+  min-width: 18.4375rem;
+  background-color: #6070ff;
+  -webkit-border-radius: 50px 0 0 0;
+  -moz-border-radius: 50px 0 0 0;
+  -ms-border-radius: 50px 0 0 0;
+  -o-border-radius: 50px 0 0 0;
+  border-radius: 50px 0 0 0;
+}
+
+.contact-footer {
+  background-image: url('images/bg-contact-me/contact-me-shapes.svg');
+  background-repeat: no-repeat;
+  background-position: right;
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+}
+
+.contact-head {
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  text-align: center;
+  margin-top: 5rem;
+}
+
+.footer-h {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 2.25rem;
+  line-height: 3rem;
+  color: #fff;
+}
+
+.footer-p {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1.15rem;
+  line-height: 1.75rem;
+  color: #ebebff;
+  padding: 1rem;
+}
+
+.footer-form {
+  display: flex;
+  flex-flow: column wrap;
+  min-width: 18.4375rem;
+}
+
+input {
+  display: flex;
+  flex-flow: column wrap;
+  align-items: stretch;
+  margin: 0.5rem 0;
+  padding: 0.75rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.75rem;
+  color: #172b4d;
+  border-style: none;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
+  border-radius: 8px;
+}
+
+textarea {
+  display: inherit;
+  justify-content: center;
+  margin: 1rem 0 2rem;
+  padding: 0.75rem;
+  min-height: 5rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1.75rem;
+  color: #172b4d;
+  border-style: none;
+  -webkit-border-radius: 8px;
+  -moz-border-radius: 8px;
+  -ms-border-radius: 8px;
+  -o-border-radius: 8px;
+  border-radius: 8px;
+}
```

Comment:
Your inputs and textarea are a bit different from the given design from the Figma template, would you fix them so that we can stick to the design as much as possible(margins, and paddings). Sticking to the design will always come in handy in your career as a software developer as clients and companies would want their sites done in a certain way.  

- your design
![Screenshot from 2022-03-29 20-46-02 (1)](https://user-images.githubusercontent.com/50773868/160675007-dcd4f6c1-c9bc-4a26-9d98-4c36f938646c.png)


-  Figma design
![Screenshot from 2022-03-29 20-46-02](https://user-images.githubusercontent.com/50773868/160675071-96f3c342-8410-48af-b033-e39ff3b87da4.png)